doc/whats_new/v0.23.rst

```diff
@@ -116,7 +116,13 @@ Changelog
 
 - |Enhancement| :func:`gaussian_process.kernels.Matern` returns the RBF kernel when ``nu=np.inf``.
   :pr:`15503` by :user:`Sam Dixon <sam-dixon>`.
-  
+
+:mod:`sklearn.impute`
+.....................
+
+- |Enhancement| :class:`impute.IterativeImputer` accepts both scalar and array-like inputs for
```

Comment:
Tell it to the user's perspective: say that we now allow max and min to be specified per feature

Reply:
Done!